From: Michal Schmidt <mschmidt@redhat.com>
Date: Fri, 3 Sep 2010 16:41:56 -0400
Subject: [net] bnx2x: move && and || to end of previous line
Message-id: <20100903164156.25415.5796.stgit@brian.englab.brq.redhat.com>
Patchwork-id: 28085
O-Subject: [RHEL5.6 BZ572012 PATCH 10/46] bnx2x: Move && and || to end of
	previous line
Bugzilla: 572012
RH-Acked-by: David S. Miller <davem@redhat.com>
RH-Acked-by: Stanislaw Gruszka <sgruszka@redhat.com>

from upstream 8e95a2026f3b43f7c3d676adaccd2de9532e8dcc

diff --git a/drivers/net/bnx2x_link.c b/drivers/net/bnx2x_link.c
index ef6a6ef..ff70be8 100644
--- a/drivers/net/bnx2x_link.c
+++ b/drivers/net/bnx2x_link.c
@@ -4716,8 +4716,8 @@ static u8 bnx2x_ext_phy_is_link_up(struct link_params *params,
 				      0xc809, &val1);
 
 			DP(NETIF_MSG_LINK, "8705 1.c809 val=0x%x\n", val1);
-			ext_phy_link_up = ((rx_sd & 0x1) && (val1 & (1<<9))
-					   && ((val1 & (1<<8)) == 0));
+			ext_phy_link_up = ((rx_sd & 0x1) && (val1 & (1<<9)) &&
+					   ((val1 & (1<<8)) == 0));
 			if (ext_phy_link_up)
 				vars->line_speed = SPEED_10000;
 			break;
